dts/bindings/memory-controllers/st,stm32-fmc-sdram.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2020, Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32 Flexible Memory Controller (SDRAM controller)
```

Comment:
Would you mind adding here an exemple of configured node ?
Now that we have a nice generated page for each binding (https://docs.zephyrproject.org/latest/reference/devicetree/bindings.html#stmicroelectronics-st), adding some info in description will add much value to the bindings documentation.
(This will have to be expanded on each binding of course).

Reply:
I have added detailed descriptions and examples to both `st,stm32-fmc` and `st,stm32-fmc-sdram`. Let me know if they are sufficient or can be improved.